draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1135,14 +1198,16 @@ The intra-domain optical links are discovered by O-PNCs, using
 mechanisms which are outside the scope of this document, and reported
 at the MPIs within the optical network topology.
 
-In case of a multi-layer DWDM/OTN network domain, multi-layer intra-domain OTN links are supported by underlay WDM tunnels: this
-relationship is reported by the mechanisms described in {{optical-path-discovery}}.
+In case of a multi-layer DWDM/OTN network domain, multi-layer
+intra-domain OTN links are supported by underlay DWDM tunnels: this
+relationship is reported by the mechanisms described in
+{{optical-path-discovery}}.
 
 {: #optical-path-discovery}
 
 ## Optical Path Discovery
 
-The WDM Tunnel Model is used to report all the WDM tunnels
+The DWDM Tunnel Model is used to report all the DWDM tunnels
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
The WDM Tunnel Model is used to report all the WDM tunnels
```